README.md

```diff
@@ -1,19 +1,27 @@
 # FFC Payment Batch Processor
 
-FFC service to process Siti Agri batch files and send individual transactions as messages.
+## Description
 
-## Prerequisites
+FFC microservice acting as an adapter Siti Agri to process batch files and send individual transactions as messages to an Azure Service Bus topic.
+Receives batch payment files, validates payment requests and then publishes the validated requests.
+
+# Prerequisites
+
+## Software to be installed
 
 - Access to an instance of an
 [Azure Service Bus](https://docs.microsoft.com/en-us/azure/service-bus-messaging/)(ASB).
+- Azure Blob Storage
```

Comment:
Should we link out to these services?

Reply:
Also, Azure Blob is same as Service Bus in that we're accessing an instance of it, not installing software, title is a little misleading, thoughts?